dashboard/src/components/AppFrame.js

```diff
@@ -4,55 +4,77 @@ import { withStyles } from "material-ui/styles";
 import { createFragmentContainer, graphql } from "react-relay";
 
 import AppRoot from "./AppRoot";
-import AppBar from "./Toolbar";
+import AppBar from "./AppBar";
 import Drawer from "./Drawer";
 import QuickNav from "./QuickNav";
 
-const styles = theme => ({
```

Comment:
Unsure why styles were moved into the class declaration, but make sure you keep in consistent in all files and tell me why the change was made (so I understand) if we keep it this way. Obviously we sometimes need certain styles in the class if the styles need to be generated a certain way depending on props, but it didn't seem like it in this case.

Reply:
In a separate task I'll back around and make the usage of withStyle, compose, createContainer and others consistent.

Reply:
Maybe make a ticket now so we don't forget